.travis.yml

```diff
@@ -32,6 +32,7 @@ install:
   - if [[ "$PYCOSAT" ]]; then
       conda install pycosat=$PYCOSAT;
     fi
+  - pip install ruamel.yaml
```

Comment:
Nice!  Thanks for pointing that out.  What projects are you aware of that use this?